systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -242,7 +242,7 @@ private static JsonNode loadConfigurationFile() {
             File jsonFile = new File(config).getAbsoluteFile();
             return mapper.readTree(jsonFile);
         } catch (IOException ex) {
-            LOGGER.info("Json configuration not provider or not exists");
+            LOGGER.info("Json configuration not provided or not exists");
```

Comment:
Should we maybe use a full sentence including some verbs? You fixed the typo, but it still sounds quite weird. Maybe `Json configuration was not provided or cannot be processed.`. It might be also good to log the exception maybe?

Reply:
Also maybe use `WARN` or `ERROR` log level makes sense

Reply:
I guess using `WARN` would be better. What do you suggest @Frawless and @scholzj ?

Reply:
Is this good as statement --> `"Json configuration was not provided or doesn't exists" ?

Reply:
TBH, I'm not sure who wrote it ... I do not follow the difference between not being provided and not existing ... isn't it the same? But I guess the `IOException` can mean also some errors at processing it.

Reply:
@ShubhamRwt ^^^?

Reply:
Thanks @scholzj for the reminder, Was doing prod work lately so this got out of my head. I will fix it by today.

Reply:
Should I use Logger.Warn ?
Since when it appeared in the system test, it didn't really broke them. it just said this is not present so it is fetching that from other place

Reply:
If it is not really required, then I think INFO level might be ok. Or should it be again separated into _not being provided_ and some _error during reading / parsing_? I do not understand the STs enough to judge whether this is error, expected situation and how often does it happen. I guess you should use `git blame` to find out who wrote this pice of code and check with them the original intention.

Reply:
Seems like it was written by David kornel. I will ping him and check if it triggers error.